training-tracker-service/src/main/java/org/tolking/training_event_service/service/impl/TrainingEventServiceImpl.java

```diff
@@ -0,0 +1,70 @@
+package org.tolking.training_event_service.service.impl;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.modelmapper.ModelMapper;
+import org.springframework.stereotype.Service;
+import org.tolking.training_event_service.dto.TrainingEventDTO;
+import org.tolking.training_event_service.entity.TrainingEvent;
+import org.tolking.training_event_service.enums.ActionType;
+import org.tolking.training_event_service.model.TrainerSummary;
+import org.tolking.training_event_service.repository.TrainerRepository;
+
+import java.time.LocalDate;
+import java.time.YearMonth;
+import java.time.ZoneId;
+import java.util.*;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class TrainingEventServiceImpl implements org.tolking.training_event_service.service.TrainingEventService {
```

Comment:
curios, if there's specific reason for full class name instead of import?

Reply:
Oh It was actually autoamtically generated interface from impl in Intellid IDEA